google/cloud/logging_v2/handlers/handlers.py

```diff
@@ -61,6 +62,12 @@ def filter(self, record):
         inferred_http, inferred_trace = get_request_data()
         if inferred_trace is not None and self.project is not None:
             inferred_trace = f"projects/{self.project}/traces/{inferred_trace}"
+        # set labels
+        user_labels = getattr(record, "labels", {})
+        record.total_labels = {**self.default_labels, **user_labels}
+        record.total_labels_str = ", ".join(
+            [f'"{k}": "{v}"' for k, v in record.total_labels.items()]
```

Comment:
just out of curiosity, would this work if users provided nested labels? actually, do we/do we not allow nested labels?